src/main/scala/Dictionary.scala

```diff
@@ -0,0 +1,5 @@
+import scala.io.Source
+
+object Dictionary {
+  val all: List[String] = Source.fromFile("src/main/resources/dictionary.txt").getLines.toList
```

Comment:
for java as well as scala this would only work if you are running locally as the source files happen to be under the root of the project.
If you are building the code into a deplyable JAR file then the dictionary would be on the classpath but not in the file system.
You can get at instead with something like 
```scala
Source.fromInputStream(getClass.getResourceAsStream("dictionary.txt"))
```